fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -301,8 +301,10 @@ private OptExpression logicalRuleRewrite(OptExpression tree, TaskContext rootTas
             }
         }
 
-        tree = new MaterializedViewRule().transform(tree, context).get(0);
-        deriveLogicalProperty(tree);
+        if (sessionVariable.isEnableSyncMaterializedViewRewrite()) {
```

Comment:
The new session variable should control we whether support directly query the sync mv, not effect the original sync mv rewrite.

Reply:
This is another thing. Add a comment, this config is only used to control whether to rewrite sync mv, so can be used in the test.